packages/compiler-dom/src/transforms/vHtml.ts

```diff
@@ -1 +1,45 @@
-// TODO
+import {
+  createStructuralDirectiveTransform,
+  createCompilerError,
+  ErrorCodes,
+  createSimpleExpression,
+  SimpleExpressionNode,
+  NodeTypes
+} from '@vue/compiler-core'
+
+export const transformHtml = createStructuralDirectiveTransform(
```

Comment:
`v-html` (and `v-text`) shouldn't be structural directive transforms. They should just be normal transforms mapping the directive to the corresponding prop.